.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
You can ignore .vscode and .idea in your global gitignore

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c